Jenkinsfile

```diff
@@ -445,25 +438,27 @@ pipeline {
           // If no tests were run then we should neither be updating commit
           // status in github nor send any slack messages
           if (currentBuild.result != null) {
-            step([
-              $class: 'GitHubCommitStatusSetter',
-              errorHandlers: [[$class: "ChangingBuildStatusErrorHandler", result: "UNSTABLE"]],
-              contextSource: [
-                $class: 'ManuallyEnteredCommitContextSource',
-                context: 'continuous-integration/jenkins/branch'
-              ],
-              statusResultSource: [
-                $class: 'ConditionalStatusResultSource',
-                results: [
-                  [$class: 'AnyBuildResult', message: 'Pipeline result', state: currentBuild.getResult()]
+            // Do not update the commit status for continuous tests
+            if (params.e2e_continuous == false) {
```

Comment:
I _think_ you still need the `currentBuild.result != null` condition as sometimes all steps are skipped but this will be setting status. Please check "empty" pipelines in https://mayastor-ci.mayadata.io/job/Mayastor/job/develop/

Reply:
I'm colorblind I guess, sorry, the condition is there still :)